fastcrypto/src/vrf.rs

```diff
@@ -239,8 +239,12 @@ pub mod ecvrf {
             let h = public_key.ecvrf_encode_to_curve(alpha_string);
 
             let challenge = RistrettoScalar::from(&self.c);
-            let u = RistrettoPoint::generator() * self.s - public_key.0 * challenge;
-            let v = h * self.s - self.gamma * challenge;
+            let u = RistrettoPoint::multiscalar_mul(
+                [self.s, -challenge],
+                [RistrettoPoint::generator(), public_key.0],
+            )
+            .unwrap();
```

Comment:
the code above
`.map(RistrettoPoint).ok_or(FastCryptoError::GeneralError)`
implies that multiscalar_mul can fail in some cases other than if the arguments have different lengths.

Reply:
Yea, that probably need a comment. It'll happen if some of the elements are None.

Reply:
Actually there's also a non-optional version that we can use instead.